keyboards/kbdfans/odinmini/info.json

```
@@ -0,0 +1,137 @@
+{
+    "manufacturer": "KBDFans",
+    "keyboard_name": "Odinmini",
+    "maintainer": "moyi4681",
+    "bootloader": "rp2040",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "GP27",
+        "num_lock": "GP4",
+        "on_state": 1
+    },
+    "matrix_pins": {
+        "cols": ["GP28", "GP26", "GP25", "GP24", "GP23", "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16", "GP0", "GP7", "GP6", "GP5", "GP3", "GP15"],
+        "rows": ["GP1", "GP2", "GP29", "GP13", "GP14"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+        "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```